types/web3-provider-engine/index.d.ts

```diff
@@ -0,0 +1,24 @@
+declare module 'web3-provider-engine' {
```

Comment:
i think this was accidentally committed (?)

Reply:
or wait never mind--it looks like this is not necessarily a compile artifact, and that you are intentionally including this to address the type errors you are running into.